src/asm/rgbasm.5

```diff
@@ -1470,6 +1470,54 @@ As in macros, you can also use the escape sequence
 .Ic \[rs]@ .
 .Ic REPT
 blocks can be nested.
+.Pp
+A common pattern is to repeat a block for each value in some range.
+.Ic FOREACH
+is simpler than
+.Ic REPT
+for that purpose. Everything
```

Comment:
Sentence-ending periods must end their line, otherwise `mandoc` does not treat it as such. (Those periods are followed by an extra space in `man` output.)

Also, did you mean "Everything *between*"?